pytorch_toolkit/object_detection/model_templates/face-detection/face-detection-0200_nncf_naive_01/template.yaml

```diff
@@ -0,0 +1,75 @@
+name: face-detection-0200_nncf_naive_01
```

Comment:
I thinkg there should not be any new template

Reply:
Agree, it is for debug purposes, will remove new templates soon